frontend/src/components/individuals/IndividualEditContent.js

```diff
@@ -18,18 +18,22 @@ const searchIndividuals = (token, input) =>
 const searchTaxons = (token, input) =>
   withToken(token, api.taxons.search)(input).then(res => res.data.results)
 
+const searchReferenceGenomes = (token, input) =>
+  withToken(token, api.referenceGenomes.search)(input).then(res => res.data.results)
```

Comment:
`api.referenceGenomes.search` doesn't seem to exist. The frontend bombs in `withToken` because that function is undefined.

Reply:
Also, is it useful to hit the backend for the search? Is this to allow the user to type in something like a genbank id and get the matching genome? My component uses the filtering built in with ant, so it only matches the assembly names listed in the menu.

Reply:
There could eventually be hundreds of reference genome which could lead to paging then the search would be useful. In here it's mostly to match the rest of the form.